pkg/daemon/node.go

```diff
@@ -17,6 +17,16 @@ const (
 	// InitialNodeAnnotationsFilePath defines the path at which it will find the node annotations it needs to set on the node once it comes up for the first time.
 	// The Machine Config Server writes the node annotations to this path.
 	InitialNodeAnnotationsFilePath = "/etc/machine-config-daemon/node-annotations.json"
+
+	// InitialConfigRequiredPath is a stamp file that signifies whether or not we have
+	// completed any "firstboot" configuration from our targeted MachienConfig.  Currently
```

Comment:
Minor: MachineConfig (unless you're talking about your dog named Config in French, in which case it's "MachienneConfig" :) )